pkg/door/door.go

```diff
@@ -0,0 +1,8 @@
+package door
+
+// Door operation contract.
```

Comment:
Abstract door operations as an interface it's more easy to create tests (mock) and materialize the platform-dependent boundary.